.github/workflows/vpm_ci.yml

```
@@ -0,0 +1,60 @@
+name: VPM CI
+
+on:
+  push:
+    paths:
+      - '**/vpm_ci.yml'
+      - 'cmd/tools/vpm.yml'
```

hm, there is no `cmd/tools/vpm.yml` file?

---

Do you mean this instead?
```suggestion
      - 'cmd/tools/vpm/**'
```

---

oops, thank you!